pkg/kine/drivers/sqlite/schema.go

```diff
@@ -61,6 +61,19 @@ CREATE TABLE kine
 	return nil
 }
 
+// applySchemaV2 moves the schema from version 1 to version 2
+func applySchemaV2(ctx context.Context, txn *sql.Tx) error {
```

Comment:
Let's make it applySchemaV0_2 and please rebase of master with my new schema version updates.